src/js/background/identityState.js

```diff
@@ -26,16 +37,57 @@ const identityState = {
       });
     },
 
-    remove(cookieStoreId) {
+    async remove(cookieStoreId) {
       const storeKey = this.getContainerStoreKey(cookieStoreId);
-      return this.area.remove([storeKey]);
+      return await this.area.remove([storeKey]);
```

Comment:
Just for reference, this has no impact. async functions just wrap the result in a promise. The return value is already a promise and is already awaited at the callsites.

Reply:
Thanks. Didn't know this!